src/loaders/common/loader_helper.py

```diff
@@ -335,7 +335,7 @@ def is_upa_info_complete(upa_dir: str):
     """
     upa = os.path.basename(upa_dir)
     fa_path = os.path.join(upa_dir, upa + ".fa")
-    meta_path = os.path.join(upa_dir, upa + ".meta")
+    meta_path = get_meta_file_path(os.path.dirname(upa_dir), upa)
```

Comment:
Is this necessary?

Reply:
you are referring to `os.path.dirname(upa_dir)`? Yes. the function gets source_dir which is the parent of upa_dir.

Reply:
👍